azkaban-common/src/test/java/azkaban/executor/ExecutorManagerTest.java

```diff
@@ -162,7 +170,8 @@ public void testSetupExecutorsException() throws ExecutorManagerException {
     Executor executor1 = loader.addExecutor("localhost", 12345);
 
     ExecutorManager manager =
-      new ExecutorManager(props, loader, new AlerterHolder(props));
+      new ExecutorManager(props, loader, new AlerterHolder(props),
```

Comment:
Is there some way that we could consolidate the code that set up an Executor Manager.